src/server/services/metadata.ts

```diff
@@ -947,75 +947,75 @@ export const translationSpecialties: QuestionData[] = [
   {
     text: "Legal",
     value: "Legal",
-    description: "e.g. property, visas, death certificates, notarised documents",
+    description: "For example property, visas, death certificates, notarised documents",
```

Comment:
should these include a comma after "For example":

from: `For example property, visas, death certificates, notarised documents`

to `For example, property, visas, death certificates, notarised documents`

Reply:
oops, bad find and replace. good spot, done